api/src/client/ApiFormationHealth.ts

```diff
@@ -122,7 +122,7 @@ export const ApiFormationHealth: UserData = {
           legalType: "foreign-limited-liability-company",
           businessName: "some-business-name-72304745",
           businessSuffix: "LLC",
-          businessStartDate: "2024-02-10",
+          businessStartDate: getCurrentDateISOString(),
```

Comment:
[pebble] Don't we want to `getCurrentDateInNewJersey()`? Would that work?

Reply:
done